src/backend/distributed/sql/citus--11.2-1--11.2-2.sql

```diff
@@ -0,0 +1,2 @@
+-- citus--11.2-1--11.2-2
+#include "udfs/worker_adjust_identity_column_seq_ranges/11.2-2.sql"
```

Comment:
I think this file (as you commented on https://github.com/citusdata/citus/pull/6820#discussion_r1157083153) exists :) Otherwise, I would expect [this test](https://github.com/citusdata/citus/pull/6820/files#diff-94fa9c57ab8e50798de70e1de4f422fb6f3138b538b9507d4f50f4083afd2485R1355) to fail. Am I missing the point?

Reply:
Sorry for my confusion. I do not know what I was thinking when I wrote this comment. The file is definitely there.